distribution/test/itests/test-itests-ddf/pom.xml

```diff
@@ -443,38 +443,29 @@
                             <asyncDestroyOnShutdown>true</asyncDestroyOnShutdown>
                         </configuration>
                     </execution>
+                </executions>
+            </plugin>
+            <plugin>
```

Comment:
❓ Should we make sure that admin, reconnect, service are all `false` just in case the defaults ever change?

Reply:
Sounds good to me.